tests/checkBoxes.spec.ts

```diff
@@ -0,0 +1,68 @@
+import { test, expect } from '@playwright/test';
+import exp from 'constants';
+
+test.beforeEach(async({page}) => {
+    await page.goto('https://petclinic.bondaracademy.com/welcome')
+})
+
+test.describe('Checkboxes', () => {
+    test.beforeEach(async ({page}) => {
+        await page.getByRole('button', {name: " Veterinarians"}).click()
+        await page.getByRole('link', {name: " All"}).click() //1. Select the VETERINARIANS menu item in the nav bar, then select "All"
+    })
+
+    test('validate selected specialties', async ({page}) => {
+        const vetDoctorTableHeader = page.locator('app-vet-list h2')
+        await expect(vetDoctorTableHeader).toHaveText('Veterinarians') //2. Assert Veterinarians text is displayed above table
+        const vetTableRowDr = page.getByRole('row', {name: " Helen Leary "})
+        await vetTableRowDr.getByRole('button', {name: 'Edit Vet'}).click() //3. Select Helen Leary and click Edit Vet
+        const specialtyDropdown = page.locator('.dropdown-display .selected-specialties') //4. Assert Specialties field is displayed with radiology
+        await expect(specialtyDropdown).toHaveText('radiology')
+        const speacialtiesDropdownArrow = page.locator('.dropdown-arrow')
+        await speacialtiesDropdownArrow.click() //5. Click on dropdown
```

Comment:
Lines 21 and 22 can be a single line command

Reply:
Changed